Build Error in Runtime (cuda_fp16.h)

```
[ 79%] Building CXX object lib/op-attrs/CMakeFiles/op-attrs.dir/src/aggregate.cc.o
cd /home/kateunger/FlexFlow/build/lib/op-attrs && /usr/bin/c++ -DFF_USE_CUDA -DFF_USE_NCCL=ON -DJSON_DIAGNOSTICS=0 -DJSON_USE_IMPLICIT_CONVERSIONS=1 -DMAX_NUM_FUSED_OPERATORS=64 -DMAX_NUM_FUSED_TENSORS=64 -DMAX_NUM_INPUTS=256 -DMAX_NUM_OUTPUTS=256 -DMAX_NUM_TASK_ARGUMENTS=5 -DMAX_NUM_TASK_REGIONS=20 -DMAX_NUM_WEIGHTS=64 -DMAX_NUM_WORKERS=1024 -DMAX_OPNAME=128 -DMAX_TENSOR_DIM=5 -Dop_attrs_EXPORTS -I/home/kateunger/FlexFlow/lib/op-attrs/src -I/home/kateunger/FlexFlow/lib/op-attrs/include -I/home/kateunger/FlexFlow/deps/variant/include -I/home/kateunger/FlexFlow/deps/optional/include -I/home/kateunger/FlexFlow/lib/utils/include -I/home/kateunger/FlexFlow/deps/expected/include -I/home/kateunger/FlexFlow/deps/visit_struct/include -I/home/kateunger/FlexFlow/deps/fmt/include -I/home/kateunger/FlexFlow/deps/rapidcheck/include -I/home/kateunger/FlexFlow/deps/invoke/headers -I/home/kateunger/FlexFlow/deps/json/single_include -I/home/kateunger/FlexFlow/deps/any -isystem /usr/include/c++/9 -isystem /usr/include/x86_64-linux-gnu/c++/9 -isystem /usr/include/c++/9/backward -isystem /usr/lib/gcc/x86_64-linux-gnu/9/include -isystem /usr/local/include -isystem /usr/include/x86_64-linux-gnu -isystem /usr/include -O3 -DNDEBUG -std=c++11 -fPIC -MD -MT lib/op-attrs/CMakeFiles/op-attrs.dir/src/aggregate.cc.o -MF CMakeFiles/op-attrs.dir/src/aggregate.cc.o.d -o CMakeFiles/op-attrs.dir/src/aggregate.cc.o -c /home/kateunger/FlexFlow/lib/op-attrs/src/aggregate.cc
In file included from /home/kateunger/FlexFlow/lib/op-attrs/include/op-attrs/datatype.h:5,
                 from /home/kateunger/FlexFlow/lib/op-attrs/include/op-attrs/parallel_tensor_shape.h:4,
                 from /home/kateunger/FlexFlow/lib/op-attrs/include/op-attrs/ops/aggregate.h:5,
                 from /home/kateunger/FlexFlow/lib/op-attrs/src/aggregate.cc:1:
/home/kateunger/FlexFlow/lib/utils/include/utils/fp16.h:5:10: fatal error: cuda_fp16.h: No such file or directory
    5 | #include <cuda_fp16.h>
      |          ^~~~~~~~~~~~~
compilation terminated.
```

--- a/cmake/cuda.cmake
+++ b/cmake/cuda.cmake
@@ -81,6 +81,9 @@ if(CUDA_FOUND)
   list(APPEND FLEXFLOW_INCLUDE_DIRS
     ${CUDA_INCLUDE_DIRS})
 
+  add_library(cuda INTERFACE)
+  target_include_directories(cuda SYSTEM INTERFACE "${CUDA_INCLUDE_DIRS}")
+
 else()
   message( FATAL_ERROR "CUDA package not found -> specify search path via CUDA_ROOT variable")
 endif()

--- a/lib/utils/CMakeLists.txt
+++ b/lib/utils/CMakeLists.txt
@@ -30,6 +30,7 @@ target_link_libraries(
   invoke
   json
   any
+  cuda
 )
 define_ff_vars(${project_target})
 ff_set_cxx_properties(${project_target})

